ui/nginx.prod.conf

```diff
@@ -13,8 +13,8 @@ http {
 
     # If we're running on App Engine, logs will appear on the Google Developer's
     # Console when logged to this directory.
-    access_log /var/log/app_engine/app.log;
-    error_log /var/log/app_engine/app.log;
+    # access_log /var/log/app_engine/app.log;
```

Comment:
Why is this commented out now? If it's not required anymore could we remove it instead?

Reply:
This line is required by the App Engine, but will break the image if you don't use the GAE and you don't override this default nginx conf file. I commented out just as a fix, but need inputs from @bfcrampton or @calbach to determine whether we want to consolidate it in this way.

Reply:
I would just remove these lines, we (dsub deploy) should be overriding the nginx file on our end, rather than cromwell doing it since the GAE is more deployment-specific.

Reply:
done!